design/product-design-list-all-accounts.md

```diff
@@ -0,0 +1,75 @@
+# LIST AND CREATE ACCOUNTS
+
+## List accounts
+
+Shows all accounts in system, has a button that allows to go to the account creation view.
+The name of each acount can be clicked and this will lead to a blank page (for the moment)
+
+## Create account
+
+Shows a form to enter the account information:
+
+- Account name: Must be unique among all other names on the system.
+- Person fullname: should be minimum 1 character long or maximun 255 characters.
+
+Finally when we press submit button, if the content provided in the form is valid, this will redirect to list accounts page, else will show an feedback error message showing form mistakes.
+
+## Wireframes
+
+Images (mocks)
```

Comment:
I like that you created the images as well as the xml files. Can we get them to show up in this markdown file by linking to them please?